Pinta.Core/Extensions/ToolBoxButton.cs

```diff
@@ -56,7 +43,7 @@ public ToolBoxButton (BaseTool tool)
 		CanFocus = false;
 
 
-		SetCssClasses (["ToolBoxButton", AdwaitaStyles.Flat]);
+		SetCssClasses (["tool-box-button", AdwaitaStyles.Flat]);
```

Comment:
It would be great to also add string constants for the style names, especially if any styles end up being used multiple times, e.g `Styles.ToolBoxButton`
Something similar to `Pinta.Resources/Icons.cs` would be a good place since the stylesheet is also embedded in that library